admin/main.js

```diff
@@ -108,26 +99,21 @@ function main() {
       password: passwordInput.value,
       userType: userTypeInput
     };
-    fetch('/admin/rpc/add-user', {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json'
-      },
-      body: JSON.stringify(data)
-    }).then((response) => {
+    postData('/admin/rpc/add-user', data).then((response) => {
       response.json().then(resp => {
         if (resp.success) {
-          location.reload(true);
+          reloadUsers();
+          refresh();
```

Comment:
We don't need to reload the whole page, reloading just the list of users would be enough.

Reply:
I like this! i didn't like the idea of reload the entire page each time